src/jdk.javadoc/share/classes/jdk/javadoc/internal/tool/resources/javadoc.properties

```diff
@@ -302,6 +302,8 @@ main.warnings.Werror=warnings found and -Werror specified
 main.unknown.error=an unknown error has occurred
 main.internal.error=an internal error has occurred
 main.unexpected.exception=an unexpected exception was caught: {0}
+main.unknown.option.for.filemanager=option not supported by file manager: {0}
+main.bad.arg.for.filemanager.option=bad value for file manager option {0}: "{1}"
 doclet.internal.report.bug=\
```

Comment:
We don't seem to have any tests for these new errors.

Reply:
Filed https://bugs.openjdk.java.net/browse/JDK-8279251